carbon-design-system/carbon
# fix: left align text in sortable header

Closes #9327 

Left aligns text in sortable header labels inside the `DataTable` component.

#### Changelog

**New**

- Add `text-align: left` to Button elements with `.bx--table-sort`

#### Testing / Reviewing

1. View the Sorting DataTable example in the Storybook
2. Resize the window until a column header splits into multiple lines
3. Check that the text is now left aligned

#### Screenshots:
**Before:**
![Screen Shot 2021-07-26 at 4 09 48 PM](https://user-images.githubusercontent.com/18495025/127059466-b5e73d96-16f8-4443-b301-8b43356ed6ac.png)

**After:**
![Screen Shot 2021-07-26 at 4 09 15 PM](https://user-images.githubusercontent.com/18495025/127059408-d9b986e0-ba9e-4efd-8e4d-c1331bc7129a.png)
